### DISCUSSION TITLE
Feature Request: When adding a "New break" it would be nice to have the option to add the break to all rooms at the same time.

### DISCUSSION THREAD

**Discussion by navlisilvan** [pretalx/pretalx#1731] [Ideas] [answered]
We have a schedule with 13 rooms.
When adding a "Lunch"-break, we have to add the break 13 times to all the 13 rooms. 
It would be great if there would be an option to add the break to all rooms at the same time.

Thanks a lot!

**Comment by rixx** [comment, marked as answer]
Tracked in #1739 (thank you, GitHub, for providing a "Create issue from discussion" button, but no backlinks), so marking as answered here.
